src/components/tree/phyloTree/grid.js

```diff
@@ -188,7 +196,7 @@ const computeYGridPoints = (ymin, ymax) => {
  */
 export const addGrid = function addGrid() {
   const layout = this.layout;
-  addSVGGroupsIfNeeded(this.groups, this.svg);
+  addSVGGroupsIfNeeded(this.groups, this.svg, this.temporalWindow);
```

Comment:
`addSVGGroupsIfNeeded` still takes only 2 arguments, but 3 are provided here

Reply:
I had tried an alternative solution which had involved modifying the signature, and accidentally left this in - good catch.